generate/generate.go

```diff
@@ -883,8 +883,27 @@ func (g *Generator) ClearProcessCapabilities() {
 	g.spec.Process.Capabilities.Ambient = []string{}
 }
 
-// AddProcessCapability adds a process capability into g.spec.Process.Capabilities.
-func (g *Generator) AddProcessCapability(c string) error {
+// AddProcessAmbientCapability adds a process capability into g.spec.Process.Capabilities.Ambient.
+func (g *Generator) AddProcessAmbientCapability(c string) error {
```

Comment:
As we did for other generator functions, function's name should be based on spec's item order. As setting value for g.spec.Process.Capabilities.Ambient, the name should be AddProcessCapabilitiesAmbient.